cleanrl/ppo.py

```diff
@@ -116,10 +121,12 @@ def __init__(self, envs):
             layer_init(nn.Linear(64, envs.single_action_space.n), std=0.01),
         )
 
-    def get_value(self, x):
+    def get_value(self, x: torch.Tensor) -> torch.Tensor:
         return self.critic(x)
 
-    def get_action_and_value(self, x, action=None):
+    def get_action_and_value(
+        self, x: torch.Tensor, action: Optional[torch.Tensor] = None
+    ) -> tuple[torch.Tensor, torch.Tensor, torch.Tensor, torch.Tensor]:
```

Comment:
Adding `from __future__ import annotations` at the top would guarantee the 3.7+ compatibility, and make dropping support later on quite easy.